ansible_collections/arista/cvp/plugins/module_utils/device_tools.py

```diff
@@ -1739,6 +1780,118 @@ def reset_device(self, user_inventory: DeviceInventory):
             results.append(result_data)
         return results
 
+    def validate_config(
+        self,
+        user_inventory: DeviceInventory,
+        validate_mode: str = ModuleOptionValues.VALIDATE_MODE_SKIP,
+    ):
+        results = []
+        device_data = {"warnings": [], "errors": []}
+        vldm_skip = ModuleOptionValues.VALIDATE_MODE_SKIP
+        vldm_err = ModuleOptionValues.VALIDATE_MODE_STOP_ON_ERROR
+        vldm_warn = ModuleOptionValues.VALIDATE_MODE_STOP_ON_WARNING
+        for device in user_inventory.devices:
+            result_data = CvApiResult(
+                action_name=device.info[self.__search_by] + "_validated"
+            )
+            if device.system_mac is not None:
+                MODULE_LOGGER.debug(
+                    "Device configlets are {0}".format(str(device.configlets))
+                )
```

Comment:
To try to avoid too much indent try to follow this kind of pattern instead:

```suggestion
            if device.system_mac is None:
                continue
            MODULE_LOGGER.debug(
                "Device configlets are {0}".format(str(device.configlets))
            )
            ...
```